packages/validator/src/services/block.ts

```diff
@@ -243,7 +247,7 @@ export class BlockProposingService {
     const debugLogCtx = {
       source: source,
       // winston logger doesn't like bigint
-      "blockValue(eth)": `${fullOrBlindedBlock.blockValue / ETH_TO_WEI}`,
+      "blockValue(approx ETH)": formatBigDecimal(fullOrBlindedBlock.blockValue, ETH_TO_WEI, MAX_DECIMAL_FACTOR),
```

Comment:
I think `approx` can be omitted, it should be obvious due to the fact that only 5 decimals are shown, this is also common practice on other websites like etherscan or beaconcha.in

would either do

```suggestion
      "blockValue(ETH)": formatBigDecimal(fullOrBlindedBlock.blockValue, ETH_TO_WEI, MAX_DECIMAL_FACTOR),
```

or  what would feel more intuitive to me

```suggestion
      "blockValue": `${formatBigDecimal(fullOrBlindedBlock.blockValue, ETH_TO_WEI, MAX_DECIMAL_FACTOR)} ETH`,
```

Reply:
thanks updated!